pkg/operator/controller.go

```diff
@@ -163,21 +162,8 @@ func (c *Controller) Handle(action string, o interface{}) {
 
 	glog.V(1).Infof("Processing %s object %s", action, objectInfo)
 
-	if cr, ok := o.(*regopapi.ImageRegistry); ok {
-		dgst, err := resource.Checksum(cr.Spec)
```

Comment:
if we're not going to look at this any more, should we remove the code that's setting it?

Reply:
done